scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -21,7 +21,7 @@
             <button class="dropdown__toggle navbtn navbtn--text-only" dropdown-toggle>{{ :: 'Monitoring' | translate }} {{workspace.name}} <span class="dropdown__caret"></span></button>
             <ul class="dropdown__menu scrollable">
                 <li><div class="dropdown__menu-label">{{ :: 'Stages' | translate }}</div></li>
-                <li ng-repeat="card in aggregate.cards" ng-class="{active: monitoring.singleGroup.">
+                <li ng-repeat="card in aggregate.cards" ng-class="{active: monitoring.singleGroup != null}">
```

Comment:
@petrjasek you added an invalid expression in the previous commit. Check out if my fix is appropriate for what you intended.

Reply:
I think this ng-class bit shouldn't be there at all

Reply:
Ok, I'll remove it then